ckan/views/api.py

```diff
@@ -0,0 +1,103 @@
+# encoding: utf-8
+
+import cgi
+import logging
+
+from flask import Blueprint, request, make_response
+
+from ckan.common import json
+
+
+log = logging.getLogger(__name__)
+
+CONTENT_TYPES = {
+    u'text': u'text/plain;charset=utf-8',
+    u'html': u'text/html;charset=utf-8',
+    u'json': u'application/json;charset=utf-8',
+}
+
+
+API_DEFAULT_VERSION = 3
+API_MAX_VERSION = 3
+
+
+# Blueprint definition
+
+api = Blueprint(u'api', __name__, url_prefix=u'/api')
+
+
+# Private methods
```

Comment:
Getting flashbacks to `#pragma` blocks in CodeWarrior :) Let's not do section comments if we can help it, the code itself should be self-describing. `_` prefix means "private", `__` "protected", a Blueprint is clearly a blueprint, etc...